### ISSUE TITLE
[Feature Request] Ability to hide specified materials (material folders) in Texture Browser

### ISSUE THREAD

**Issue by nikita488** [ficool2/HammerPlusPlus-Issue-Tracker#64]
There are a lot of textures that won't be used on brushes (icons, sprites, decals, etc), so the ability to hide them will help a lot to reduce the number of textures to browse and improve workflow. Probably with a checkbox like "Show hidden textures" so that if there is a need to see _all_ textures, we can toggle it.

**Comment by SethTooQuick** [comment]
You can already hide folders natively in Hammer. However, it would be useful if individual textures could be hidden too when specifying the entire file name.

**Comment by nikita488** [comment]
Yes, we can, but it only works with unpacked content. I can add folders in the Tools - Options - Materials exclusion list, but many people mount vpks directly and there is no way to hide materials in this case iirc.

**Comment by SethTooQuick** [comment]
Doesn't sound right. It should be excluding that path, no matter if it's in a VPK or not.

**Comment by SethTooQuick** [comment]
I exclude VGUI folders and Skybox in my settings, and those are in CSGOs VPK.
